app/models/form526_submission.rb

```diff
@@ -281,13 +283,12 @@ def all_other_jobs_succeeded_if_any?
   # @param first_name [String] the first name of the user that submitted Form526
   # @return [String] the workflow batch id
   #
-  def perform_ancillary_jobs(first_name)
+  def perform_ancillary_jobs
     workflow_batch = Sidekiq::Batch.new
     workflow_batch.on(
       :success,
       'Form526Submission#workflow_complete_handler',
-      'submission_id' => id,
-      'first_name' => first_name
```

Comment:
Don't need to pass `first_name` argument; `get_first_name` can be called when it is needed.